vncviewer/CMakeLists.txt

```diff
@@ -100,8 +100,20 @@ if(UNIX)
       DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/vncviewer.desktop.in
     )
   endif()
-  add_custom_target(desktop ALL DEPENDS vncviewer.desktop)
-  install(FILES ${CMAKE_CURRENT_BINARY_DIR}/vncviewer.desktop DESTINATION ${CMAKE_INSTALL_FULL_DATADIR}/applications)
+  # FIXME: Add translations to org.tigervnc.vncviewer.metainfo.xml
```

Comment:
Can we get this sorted? Does `intltool` and `xgettext` have support for what we need?

Reply:
I am not familiar with intltool and xgettext. I may take a look later but not today.

Reply:
Translations are now supported using either xgettext or intltool.